src/actions/actionCreators.js

```diff
@@ -447,3 +458,27 @@ export const setNewCategory = (category) => ({
   type: SET_NEW_CATEGORY,
   payload: { category },
 });
+
+export const setClaimVisibility = (claim, value) => ({
+  type: SET_CLAIM_VISIBILITY,
+  payload: { claim, value },
+});
+
+export const updateSelectedClaims = (claim) => ({
```

Comment:
If you name it in plural, you would expect that the action creator takes an array of claims as a parameter and not one claim.